aurora-mysql-plugin/pom.xml

```diff
@@ -79,40 +85,4 @@
       <scope>compile</scope>
     </dependency>
   </dependencies>
-
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.felix</groupId>
-        <artifactId>maven-bundle-plugin</artifactId>
-        <version>3.3.0</version>
-        <extensions>true</extensions>
-        <configuration>
-          <instructions>
-            <_exportcontents>
-              io.cdap.plugin.*;
-              org.apache.commons.lang;
```

Comment:
Not related to this work, but these exports seem problematic. Have opened PLUGIN-710 to track it.

We should try to run a pipeline that reads from two different dbs (eg: mysql and sqlserver) and see if it fails. If not, it's something that we can address later.